### PR TITLE
Allow publishing of NS projects to iTunes

### PR DESCRIPTION
**Pull request by rosen-vladimirov**
Allow publishing of NS projects to iTunes Connect.
http://teampulse.telerik.com/view#item/286402

### CODE REVIEW

File: lib/commands/itunes-connect.ts [hunk 1]
```diff
@@ -97,16 +97,20 @@ export class UploadApplicationCommand extends AppstoreApplicationCommandBase {
...
-    
+
 	allowedParameters = [this.$stringParameterBuilder.createMandatoryParameter("No application specified. Specify an application that is ready for upload in iTunes Connect."),
 		new commandParams.StringCommandParameter(this.$injector), new commandParams.StringCommandParameter(this.$injector)];
-        
+
 	execute(args:string[]): IFuture<void> {
 		return (() => {
-			this.$project.ensureCordovaProject();
+			if(this.$project.projectData.Framework === this.$projectConstants.TARGET_FRAMEWORK_IDENTIFIERS.MobileWebsite) {
```
Comment (Fatme) [inline lib/commands/itunes-connect.ts]: This error should be thrown from `web-site-project.ts`

Comment (Fatme) [inline lib/commands/itunes-connect.ts]: I suggest you to add `canUploadToAppstore` to project `capabilities` hash.

**Comment by teobugslayer** [comment]
:+1:

**Comment by Fatme** [comment]
:+1:

**Pull request merged**